### ISSUE TITLE
DND Kit doesnt work after collapsible/accordion open and close

### ISSUE THREAD

**Issue by kasvith** [clauderic/dnd-kit#990]
**Problem**:

- We need to organize some items using collapsible. 
- Each collapsible need to be ordered(so we put them inside `<SortableContext />`)
- Items inside the collapsible(body) also has own `<SortableContext />` to sort internal content. They also contain `useDroppable`, `useDraggable` hooks
- Items can be dragged to other collapsible items
- We have an outer DND Context which handles all events and arrange elements with appropriate approach
- We also have a `DragOverlay` which uses a portal

Everything works fine until we close a collapse. Then nothing works correctly
(We are using antdesign, but i tried using `headlessui` also. Same result)

We arent using `useSortable` inside `DragOverlay`

What might be the issue?

```
<DndContext>
    // handle all events
    <SortableContext>
        // sort sections
        <Section>
            // we have useDroppable, useDraggable with two div elements.
            <div> // droppable
                <div> // draggable
                    <SortableContext>
                        ... internal items for sorting
                    </SortableContext>
                </div>
            </div>
        </Section>
        <Section></Section>
        <Section></Section>
    </SortableContext>

    <Portal>
        <DragOverlay></DragOverlay>
    </Portal>
</DndContext>
```

https://user-images.githubusercontent.com/13379595/210964553-2ac17901-6fe8-4f36-9e85-b700f5aa8f7d.mp4

**Comment by vldnovikov** [comment]
@kasvith  did you set the min-height for the last droppable? it looks like it has 0 height

**Comment by kasvith** [comment]
@vldnovikov no, droppable is even outside the collapse element. I checked the height is always there

**Comment by kasvith** [comment]
I updated example and provided a codesandbox

**Comment by pavogq** [comment]
@kasvith did you manage to find a fix for this? I'm having the same issue with headless ui and it's disclosure component.

**Comment by kasvith** [comment]
No luck

**Comment by Voxz-ai** [comment]
@kasvith did you happen to find a solution to this? i am running into the same issue.

**Comment by kasvith** [comment]
@Voxz-ai i changed the UX avoiding the collapsibles

**Comment by Voxz-ai** [comment]
For anyone else reading this, i was able to get it to work using MUI collapsible. If you are still working on it you should give that a try. 

Cheers!

**Comment by clauderic** [comment]
This issue was opened against a previous version of dnd-kit. The library has since undergone a major rewrite.

It has been rebuilt from the ground up with support for **multiple frameworks** (React, Vue, Svelte, Solid, and vanilla JavaScript), improved performance, and a new API.

- **Documentation**: https://dndkit.com
- **Migration guide**: https://dndkit.com/react/migration
- **GitHub**: https://github.com/clauderic/dnd-kit

If this issue is still relevant to the latest version, please open a new issue.
